pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "visier-connector-python"
+authors = [
+    {name = "Visier Research & Development", email = "info@visier.com"},
+]
+description = """Visier People Data connector"""
```

Comment:
This is different from the previous description (Visier People Data connector through the Visier SQL-like API). Is this intentional?

Reply:
Yes, the connector is more generic than that description would suggest